prow/apis/prowjobs/v1/types.go

```diff
@@ -161,6 +161,13 @@ type ProwJobSpec struct {
 
 	// ReporterConfig holds reporter-specific configuration
 	ReporterConfig *ReporterConfig `json:"reporter_config,omitempty"`
+
+	// Secret specifies if the Job is considered secret
```

Comment:
This could use an explicit clarification of what the effects of being "considered secret" are?

Reply:
Done